cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -136,6 +138,7 @@ protected boolean removeEldestEntry(Map.Entry<UUID, UserTaskInfo> eldest) {
     }
     _sessionExpiryMs = sessionExpiryMs;
     _maxActiveUserTasks = maxActiveUserTasks;
+    _apiUrlPrefix = KafkaCruiseControlServletUtils.REQUEST_URI;
```

Comment:
`REQUEST_URI` is a malformed `_apiUrlPrefix` -- i.e. it does not end with `*` -- hence, it shall not be used interchangeably here.

With the proposed change in `ParameterUtils` (please see earlier comment), we can drop these changes.

Reply:
I changed it to:
```
_apiUrlPrefix = config.getString(WebServerConfig.WEBSERVER_API_URLPREFIX_CONFIG);
```